src/components/auth/LoginContainer.tsx

```diff
@@ -35,6 +36,11 @@ const mapDispatchToProps = (dispatch: any) => ({
   onLogin: (providerType: ProviderType | null) => {
     track('click_login_button', { provider_type: providerType || 'guest' })
     authenticate(providerType)
+  },
+  onCancelLogin: () => {
+    track('click_login_button')
```

Comment:
is it okay that this is tracking the same event as the login? shouldn't be a "cancel login" event instead?

Reply:
should be different